client/dashboard/components/collapsible-card/index.tsx

```diff
@@ -56,22 +60,24 @@ export const CollapsibleCard = ( {
 			isBorderless={ isBorderless }
 		>
 			<CardBody>
-				<HStack justify="space-between" className="collapsible-card__header">
-					{ header }
-					<Button
-						icon={ chevronUp }
-						className={ clsx( 'collapsible-card__toggle', { collapsed: ! isExpanded } ) }
-						onClick={ handleCollapsedChange }
-						aria-expanded={ isExpanded }
-						aria-controls={ id }
-						aria-label={ label }
-					/>
-				</HStack>
-				{ isExpanded && (
-					<div className="collapsible-card__content" id={ id }>
-						{ children }
-					</div>
-				) }
+				<VStack spacing={ 2 }>
```

Comment:
Should we move this out into a separate PR to ensure it's tested against all CollapsibleCard use cases?